### PR TITLE
Quick-start for ranking - Added --shuffled_train arg to ranking.py script

### PR DESCRIPTION
**Pull request by gabrielspmoreira**
- Added --shuffled_train arg to ranking.py script
- Making --train_metrics_steps 1 by default (instead of 10) because it seems that configuration is also affecting the model.evaluate()
- Setting train shuffling correctly in the Loader, and not in model.fit()
- Set drop_last=True in fit()
- Disabled validation metrics (validation_data) during training

### CODE REVIEW

File: examples/quick_start/scripts/ranking/ranking.py [hunk 1]
```diff
@@ -251,7 +251,7 @@ def train_eval_stl(self, model):
                 epochs=self.args.epochs,
                 batch_size=self.args.train_batch_size,
                 steps_per_epoch=self.args.train_steps_per_epoch,
-                shuffle=False,
+                shuffle=self.args.shuffled_train,
```
Comment (rnyak) [inline examples/quick_start/scripts/ranking/ranking.py:254]: @gabrielspmoreira I am curious if shuffle arg in model.fit works as expected? meaning does it really shuffle data or not, if it is not explicitly defined in the Loader? thanks.

Comment (gabrielspmoreira) [reply examples/quick_start/scripts/ranking/ranking.py:254]: I think you are right. Shuffle here only applies for Dataset. Let me change it to set shuffle in the loader
